.github/workflows/linux.yml

```diff
@@ -55,8 +55,16 @@ jobs:
     runs-on: ubuntu-20.04
     steps:
       - uses: actions/checkout@v3
+        if: github.event_name != 'issue_comment'
         with:
           fetch-depth: 2
+      # This is necessary to checkout the pull request if this run was triggered
+      # via an `issue_comment` action on a pull request.
+      - uses: actions/checkout@v3
+        if: github.event_name == 'issue_comment'
```

Comment:
Any particular reason for not including the `&& github.event.issue.pull_request` constraint like its is done in `linux-wpt.yml`?

Reply:
I meant to make all of the if block symmetric, so to avoid a case where a checkout attempt wasn't made. I missed this one though. I've pushed a new version of the branch that fixes it.

Reply:
I see mac.yml still has '&& github.event.issue.pull_request' in the if block. Maybe that can also be removed?